firefox/MozillaFirefox.spec

```diff
@@ -234,6 +256,8 @@ Development files for %{appname} to make packaging of addons easier.
 Summary:        Common translations for %{appname}
 Group:          System/Localization
 Provides:       locale(%{name}:ar;ca;cs;da;de;el;en_GB;es_AR;es_CL;es_ES;fi;fr;hu;it;ja;ko;nb_NO;nl;pl;pt_BR;pt_PT;ru;sv_SE;zh_CN;zh_TW)
+# This is there for updates from Firefox before the translations-package was split up into 2 packages
+Provides:       %{name}-translations
```

Comment:
I cannot even remember the time when this was the case. Is this really still relevant? Doesn't really hurt but ...

Reply:
This was removed some time ago and promptly caused a bug to pop up saying this is broken. So some customers seem to still upgrade from ancient versions.

Reply:
ok